### DISCUSSION TITLE
[Windows] Multiple root or drive

### DISCUSSION THREAD

**Discussion by ZenomeFR** [filebrowser/filebrowser#3517] [General]
Hi :)
I'm new to filebrowser, and firstly I want to thanks all devs working on it !

I have setup my filebrowser instance using the Windows version and I'm trying to access multiple hard drive but I can't figure how to...

Here's my config.json

`{
  "port": myport,
  "address": "my.local.ip",
  "log": "stdout",
  "database": "filebrowser.db",
  "root": "D:/"
}
`

How can I manage to also mount my C drive ?

Thanks a lot for the help :)
Zenome

**Comment by Nighted** [comment]
I just created a directory inside a directory I created for File Browser: File Browser > Nighted. Where is says "scope" under user management I put Nighted.

Inside Nighted I created symbolic links to the directories I wanted access too.

When I access File Browser from my browser I see whatever stuff I have in the directory and my added links. I can click those links and they will open as expected. One is on a different drive. No problems. If you want an easy way to create the links I suggest Link Shell Extension. Right click on the place you want to link to then go to File Browser and drop the link. Super easy, much simpler than junction from Sysinternals.

It's almost been a year. Shame no one else bothered to reply until now. Hope you can still use this information.

**Reply by mylegitches** [reply]
Gonna try this.  Thanks.
